CHANGELOG.md

```diff
@@ -51,6 +41,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - New bands added to the [common band names](extensions/eo/README.md#common-band-names) for the EO extension: yellow, rededge, and 2 narrow NIR bands
 - [Scientific extension](extensions/scientific/README.md) can be used in Collections.
 
+### Fixed
+- Updated language, fixed typos and examples.
+- [Label extension](extensions/label/README.md):
```

Comment:
The label extension was just introduced in 0.8.0-rc1. Therefore the changelog entries should be removed or be marked with a respective note. Otherwise it looks like it was a change between 0.7 and 0.8, which it wasn't.